symphony/bdk/core/service/presence/presence_service.py

```diff
@@ -36,10 +39,12 @@ class OboPresenceService:
     * Delete a created presence feed
     """
 
-    def __init__(self, presence_api: PresenceApi, auth_session: AuthSession):
+    def __init__(self, presence_api: PresenceApi, auth_session: AuthSession, retry_config: BdkRetryConfig):
         self._presence_api = presence_api
         self._auth_session = auth_session
+        self._retry_config = retry_config
```

Comment:
BTW how is the retry config passed to `refresh_session_if_unauthorized` ?

Reply:
The @retry decorator fetches the `_retry_config` attribute in a specific class to update the retry parameters.
It's done in the retry custom decorator in `retry/__init__.py`